### PR TITLE
Fixing misalignment architecture components table cells

### LINKED ISSUE #7164: [UI] - Architecture Components table misaligned cells
#### Current Behavior
Some cells are misaligned

#### Desired Behavior
All cells are aligned to the same pattern 

#### Screenshots/Logs

Screenshot 2023-02-23 130113
https://docs.meshery.io/concepts/architecture
#### Environment

- Browser: Edge
- Host OS: Windows

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Anand-Theertha**
**Notes for Reviewers**

This PR fixes #7164. The PR also redirects to the correct repository (meshery/meshery instead of layer5io/meshery) in '[Contributing to Meshery Docs' ](https://docs.meshery.io/project/contributing/contributing-docs#get-the-code) section. 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f4ac01ee38c4e86e783ac050c6cdeaab7a272f4e at: https://63f788b07f0e8b2562ab1cfe--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit af552a12ab45966ed519b90aeecad0652658a205 at: https://63f83d52f6a6d31f94e0f04b--meshery-docs-preview.netlify.app

**Comment by Anand-Theertha** [comment]
Added another fix on the same PR - 
`rbenv install --list-all `command has been updated to `rbenv install --list` in "contributing to meshery docs" page.

File: docs/pages/concepts/architecture/architecture.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Meshery and its components are written using the following languages and technol
 | Meshery Provider UI                                                  | ReactJS, NextJS                                                                   |
 | [Meshery Remote Providers](/extensibility/providers)                 | _any_ - must adhere to Meshery [Extension Points]({{site.baseurl}}/extensibility) |
 | [Meshery Operator](/concepts/architecture/operator)                  | Golang                                                                            |
-| &nbsp;&nbsp;&nbsp;&nbsp; [MeshSync](/concepts/architecture/meshsync) | Golang                                                                            |
-| &nbsp;&nbsp;&nbsp;&nbsp; [Broker](/concepts/architecture/broker)     | Golang, NATS                                                                      |
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/architecture.md:26]: These are sub-components and meant to be indented.

File: docs/pages/project/contributing/contributing-docs.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libre
 
 ### Get the code
 
-- Fork and then clone the [Meshery repository](https://github.com/layer5io/meshery)
+- Fork and then clone the [Meshery repository](https://github.com/meshery/meshery)
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:114]: Good

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @Anand-Theertha.

Some feedback.

File: docs/pages/concepts/architecture/architecture.md (hunk 1 above)
Comment (Anand-Theertha) [reply docs/pages/concepts/architecture/architecture.md:26]: Uh oh. The issue asked me to do it, so I went ahead and did it hahah. Sorry, I should have thought this through!

**Comment by l5io** [comment]
🚀 Preview for commit c2f48c9cacabc218ff7877361266e9ed48f3934d at: https://63f982e8c0c8254148af416c--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]

**Pull request merged**